### DISCUSSION TITLE
Blowfish with self hosted Firebase alternative like appwrite?

### DISCUSSION THREAD

**Discussion by cbirkenbeul** [nunocoracao/blowfish#1827] [General]
Hello,
anyone tested or runs the blowfish theme with a self hosted alternative to firebase? I would not like use google services for this site but the features like the like counter seems interesting.
Thanks.

**Comment by john1-design** [comment]
I use github. It works like a charm.
Thank-you,

John De Silva
Georgetown, Ontario
Cell: 905-399-7203

**Reply by cbirkenbeul** [reply]
Do you host your site also on GitHub?
